README.md

```diff
@@ -1 +1,25 @@
-# snyk-code-local-engine-changelog
\ No newline at end of file
+# Snyk Code Local Engine Public Repository
+
+Welcome to the Snyk Code Local Engine public repository, where you can find essential information about Snyk Code Local Engine for our customers.
+
+## Overview
+
+This repository serves as a centralized source of information for our customers, providing access to the following resources:
+
+- **Changelog:** Stay updated with the latest changes, enhancements, and bug fixes made in each version of Snyk Code Local Engine. The changelog provides a detailed history of modifications to help you understand the evolution of the product.
+
+- **Public Docs:** Access comprehensive documentation that covers various aspects of Snyk Code Local Engine, including installation guides, usage instructions, and best practices. Our public documentation is designed to assist you in effectively utilizing the features of Snyk Code Local Engine.
+
+- **Values File:** Explore the relevant values file specific to the version you are using. These files contain essential configuration settings and parameters tailored to different versions of Snyk Code Local Engine.
+
+## Navigating Versions
+
+To find information related to a specific version of Snyk Code Local Engine, you can use Git tags to navigate through the repository. Each tagged version represents a distinct release of the product. By checking out the desired tag, you can access the relevant files, including the changelog and corresponding value files, providing you with the accurate and version-specific details you need.
+
+### Using Git Tags
```

Comment:
Assuming that this can be easily consumed directly on GitHub as well, I think we should add a link to:
https://github.com/snyk/snyk-code-local-engine-public/tags